client/post-editor/editor-media-modal/index.jsx

```diff
@@ -72,10 +72,9 @@ class EditorMediaModal extends Component {
 
 	onClose = value => {
 		if ( value ) {
-			this.insertMedia( value );
-		} else {
-			this.props.onClose();
+			return this.props.isGutenberg ? this.props.onClose( value ) : this.insertMedia( value );
```

Comment:
It is not clear to me what's going on here. I assume this change is needed for having `EditorMediaModal` working in both contexts (with normal blocks and classic blocks). Maybe you can add a comment in the code providing more info?